CHARTER.md

```diff
@@ -0,0 +1,237 @@
+
+Secondary Concerns
+- Hypermedia
+- Generating JSON Schema
+- Using JSON Schema to generate
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+Section Intentionally Left Blank
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the JSON Schema project, it is delegated to the JSON Schema Technical Steering Committee ("TSC").
+
+### 2.1 Other Formal Project Relationships
+Section Intentionally Left Blank
+
+## Section 3: JSON Schema Org Governing Body (TSC)
+The TSC is initially established from the observed major contributors who are currently active and in good standing.
+
+There is no maximum TSC membership size. The TSC must have a minimum of four members.
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+TSC memberships are not time-limited.
+
+While the project is not looking to obtain "Impact" project status within the OpenJS Foundation, there is no requirement set out to limit the number of TSC members by employer. It is in some cases considered difficult or even unhelpful for the project to limit the number or percentage of TSC members by employer (Especially when an employer has employed individuals already active in the community to work exclusively on the open source project). While at this time there are no limits on TSC membership by employer, the TSC will strive to keep to at least less than 50%, ideally 33% (One third, one in three). The TSC will re-evaluate this specific clause at least every six months, and aim to revise it within one year to meet the "no more than 1/3 employer member affiliation" mandate.
+
+TSC members are expected to regularly participate in TSC activities.
+
+The TSC will meet regularly using virtual conferencing tools. The meeting will be directed by the TSC Chairperson(s). Responsibility for directing individual meetings may be delegated by a TSC Chairperson to any other TSC member. Minutes or an appropriate recording will be taken and made available to the community through accessible public postings.
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
+
+A TSC member may be removed by vote from the TSC if, during a 3-month period, all of the following are true:
+
+- They do not participate in any TSC related discussions or votes
+- They do not provide any form of excuse or no excuse is known for their absence
+
+The initial TSC members are @relequestual, @awwright, @gregsdennis, @Julian, @jdesrosiers, @karenetheridge, with @relequestual being the initial chair.
```

Comment:
Since this is an official document, maybe we should specify actual names instead of github handles.

```suggestion
The initial TSC members are [Ben Hutton](https://github.com/relequestual), [Austin Wright](https://github.com/awwright), [Greg Dennis](https://github.com/gregsdennis), [Julian Berman](https://github.com/Julian), [Jason Desrosiers](https://github.com/jdesrosiers), and [Karen Etheridge](https://github.com/karenetheridge), with Ben Hutton being the initial chair.
```